dependencyManagement/build.gradle.kts

```diff
@@ -5,8 +5,9 @@ plugins {
 data class DependencySet(val group: String, val version: String, val modules: List<String>)
 
 val DEPENDENCY_BOMS = listOf(
-        "com.fasterxml.jackson:jackson-bom:2.11.0",
-        "org.junit:junit-bom:5.6.2"
+        "com.amazonaws:aws-java-sdk-bom:1.12.228",
+        "com.fasterxml.jackson:jackson-bom:2.12.0",
```

Comment:
I wanted to remove this `jackson` reference and the one below, but it failed to find them at `compileOnly` time. I was hoping it would get it through the `com.amazonaws:aws-java-sdk-bom` dependency which requires them.

However, now my theory is that it won't see them because `com.amazonaws:aws-java-sdk-bom` doesn't import them in a way that is visible to downstream packages. So we will have to continue updating it for future AWS Java SDK updates.

Reply:
v1.12.x shadows Jackson dependencies since customers were relying on the version that came with the SDK, preventing the team from upgrading from insecure versions to newer ones without introducing breaking changes.

Hence the change in v1.12.x which forces customers to bring their own Jackson version.